presto-hive/src/main/java/io/prestosql/plugin/hive/HiveAnalyzeProperties.java

```diff
@@ -62,7 +74,7 @@ public HiveAnalyzeProperties()
     public static Optional<List<List<String>>> getPartitionList(Map<String, Object> properties)
     {
         List<List<String>> partitions = (List<List<String>>) properties.get(PARTITIONS_PROPERTY);
-        return partitions == null ? Optional.empty() : Optional.of(partitions);
+        return Optional.ofNullable(partitions);
```

Comment:
this change is part of `Allow specifying a subset of columns in ANALYZE` commit, but should belong to `Simplify the way partition lists are decoded from ANALYZE properties` commit